src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(int length, int width, String color) {
```

Comment:
Creating a rectangle, avoid passing expressions directly in the constructor. For better readability, assign the random values to separate variables before passing them to the constructor.